src/main/java/no/nav/pto/veilarbportefolje/aktiviteter/AktivitetService.java

```diff
@@ -27,16 +30,42 @@ public class AktivitetService extends KafkaCommonConsumerService<KafkaAktivitetM
     private final OppfolgingsbrukerRepositoryV3 oppfolgingsbrukerRepository;
     private final SisteEndringService sisteEndringService;
     private final OpensearchIndexer opensearchIndexer;
+    private final TiltakService tiltakService;
+    private final UnleashService unleashService;
 
     public void behandleKafkaMeldingLogikk(KafkaAktivitetMelding aktivitetData) {
         AktorId aktorId = AktorId.of(aktivitetData.getAktorId());
         sisteEndringService.behandleAktivitet(aktivitetData);
+
```

Comment:
These changes should not be here, I guess you have to pull from dev and merge it into your PR